src/appReview/saga.ts

```diff
@@ -18,9 +23,17 @@ export function* setAppReview() {
 
   // If the last interaction was less than a quarter year ago or null
   if (lastInteractionTimestamp <= now - ONE_DAY_IN_MILLIS * 91) {
-    const result = yield call(InAppReview.RequestInAppReview)
-    // If the in app review was shown, update the last interaction timestamp
-    if (result) yield put(setLastInteractionTimestamp(now))
+    try {
+      const result = yield call(InAppReview.RequestInAppReview)
+      // If the in app review was shown, update the last interaction timestamp
+      if (result) {
```

Comment:
this made me wonder what to do when `result` is false...but reading the docs, it seems like `result` is always true for iOS, and true for Android _after_ they finish the review flow. am i understanding that right? if so, it seems like we should remove this if block - we always want to update the timestamp and dispatch the impression event

Reply:
It's not quite clear to me from the docs if result is always true for iOS and Android or an error. In [this example](https://github.com/MinaSamir11/react-native-in-app-review#usage) it looks like both was are used, but I also see that the updating timestamp is occurring outside the if block.

Reply:
After looking a bit more I think we're safe to remove the it block as result looks like it's either: `true` or an error.
- [react-native-in-app-review Android](https://github.com/MinaSamir11/react-native-in-app-review/blob/master/android/src/main/java/com/ibits/react_native_in_app_review/AppReviewModule.java#L68-L72)
- [react-native-in-app-review iOS](https://github.com/MinaSamir11/react-native-in-app-review/blob/master/ios/RNInAppReviewIOS.swift#L9C2-L23)

Reply:
Fixed in 3544d91aff763ed658c51a9bffa57f7521f1424d!